tests/integration/CMakeLists.txt

```diff
@@ -18,7 +18,8 @@
 cmake_minimum_required(VERSION 3.5)
 
 set(OLP_SDK_INTEGRATIONAL_TESTS_SOURCES
-    ./placeholder.cpp
+    ./olp-cpp-sdk-authentication/TestAuthenticationClient.cpp
```

Comment:
Please also rename them -> AuthenticationClientTest.cpp
`Test` should be at the end.